data_collection/web/javascript_tracking_client/api.rst

```diff
@@ -2407,12 +2407,11 @@ JavaScript Tracking Client configuration
 .. function:: setApiUrl(url)
 
     Overrides HTTP API URL for tracking endpoint that was set at the tracker initiation.
-    Usually there is no need to use this function since standard setup should use correct URL.
+    It's outdated function since standard setup should use correct URL.
 
     .. deprecated:: 16.17
 
-        Use :func:`setTrackerUrl` instead. Previously this method was useful when "Overlay" report didn't work, but that
-        report was removed some time ago.
+        This method is outdated, use :func:`setTrackerUrl` instead.
```

Comment:
Two small nitpicks. In most places we use term "function" instead of "method". Why "outdated" and not "deprecated"?

Reply:
It's already described in deprecated notice and we lost use case for this function (Overlay report for which it was meant for was removed/replaced with new Analytics report engine).